ONSdigital/rm-sample-service
# Added state diagrams for sample unit and sample summary

# Motivation and Context
This PR adds state diagrams for sample units and sample summaries and the means to build them

# What has changed
No code has been changed but a Makefile has been added

# How to test?
```
make diagrams
/Applications/Google\ Chrome.app/Contents/MacOS/Google\ Chrome diagrams/sample-unit-state.svg
/Applications/Google\ Chrome.app/Contents/MacOS/Google\ Chrome diagrams/sample-summary-state.svg
```
Ensure diagrams can be viewed
